domain/src/main/java/com/example/domain/DummyClass.kt

```diff
@@ -0,0 +1,3 @@
+package com.example.domain
+
+class DummyClass()
```

Comment:
Please remove this class and instead indicate on the PR description how we can test the integration.

Reply:
Done